### ISSUE TITLE
via fine-tune

### ISSUE THREAD

**Issue by 1144181135** [microsoft/Oscar#88]
what is the "--model_name_or_path pretrained_models/base-vg-labels/ep_107_1192087"? I didn't fine such a file.

**Comment by jo6gj94** [comment]
you can find on 

>https://github.com/microsoft/Oscar/blob/master/DOWNLOAD.md

this website, and use 

> wget https://biglmdiag.blob.core.windows.net/oscar/pretrained_models/$MODEL_NAME.zip

to download what you need .

In your case you need to use

> wget https://biglmdiag.blob.core.windows.net/oscar/pretrained_models/base-vg-labels.zip

this commend to download the base-vg-labels.zip that what you need.
